analyzer_test.go

```diff
@@ -5,15 +5,14 @@
 package dep
 
 import (
-	"io/ioutil"
 	"os"
 	"path/filepath"
 	"testing"
 
 	"github.com/golang/dep/test"
 )
 
-func TestDeriveManifestAndLock(t *testing.T) {
+func TestAnalyzerDeriveManifestAndLock(t *testing.T) {
```

Comment:
I added "Analyzer" to the test name to make it easier to run it with go test -run Analyzer.